test/e2e/isFocusable.cy.js

```diff
@@ -428,9 +429,9 @@ describe('isFocusable', () => {
         container.innerHTML = fixtures.inert;
         document.body.append(container);
 
-        for (const child of container.children) {
-          expect(isFocusable(child)).to.eql(false);
-        }
+        const focusables = Array.from(container.children).filter(isFocusable);
+
+        expect(getIdsFromElementsArray(focusables)).to.eql([]);
```

Comment:
Not related to the change, but it makes debugging easier (you get a complete list of `id`s in the diff, instead of a single true/false diff). Other tests also show lists ids in the diff, so it seemed more consistent.